CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ If you would like to contribute enhancements or fixes, please do the following:
 Please note that modifications should follow these coding guidelines:
 
 - Indent is 2 spaces.
-- Code should pass `coffeelint` linter.
+- Code should pass `eslint` linter (run `npm run-script lint`).
```

Comment:
Nah, it's fine to leave it as is. I actually didn't know that was the full command and looked it up 😛.